validator/sawtooth_validator/gossip/gossip.py

```diff
@@ -680,27 +714,51 @@ def retry_static_peering(self):
                                     static_peer_info.retry_threshold * 2,
                                     MAXIMUM_STATIC_RETRY_FREQUENCY),
                                 count=static_peer_info.count + 1)
-                    else:
-                        self._static_peer_status[endpoint] = \
-                            StaticPeerInfo(
-                                time=time.time(),
-                                retry_threshold=min(
-                                    static_peer_info.retry_threshold * 2,
-                                    MAXIMUM_STATIC_RETRY_FREQUENCY),
-                                count=0)
+                else:
+                    self._static_peer_status[endpoint] = \
+                        StaticPeerInfo(
+                            time=time.time(),
+                            retry_threshold=min(
+                                static_peer_info.retry_threshold * 2,
+                                MAXIMUM_STATIC_RETRY_FREQUENCY),
+                            count=static_peer_info.count)
 
-                    LOGGER.debug("attempting to peer with %s", endpoint)
-                    self._network.add_outbound_connection(endpoint)
-                    self._temp_endpoints[endpoint] = EndpointInfo(
-                        EndpointStatus.PEERING,
-                        time.time(),
-                        INITIAL_RETRY_FREQUENCY)
+                LOGGER.debug("attempting to peer with %s", endpoint)
+
+
+<< << << < HEAD
```

Comment:
@CAGS295 There's merge conflict tags here, which break the build:
> sawtooth-validator | [2021-10-03 00:04:01.635 WARNING  (unknown file)] [src/pylogger.rs: 39] Started logger at level DEBUG
sawtooth-validator | [2021-10-03 00:04:01.813 DEBUG    ffi] loading library libsawtooth_validator.so
sawtooth-validator | [2021-10-03 00:04:01.815 DEBUG    ffi] loading library libsawtooth_validator.so
sawtooth-validator | [2021-10-03 00:04:01.971 ERROR    (unknown file)] failed to load sawtooth_validator.server.cli
sawtooth-validator | Traceback (most recent call last):
sawtooth-validator |   File "/usr/lib/python3/dist-packages/sawtooth_validator/server/cli.py", line 32, in <module>
sawtooth-validator |     from sawtooth_validator.server.core import Validator
sawtooth-validator |   File "/usr/lib/python3/dist-packages/sawtooth_validator/server/core.py", line 61, in <module>
sawtooth-validator |     from sawtooth_validator.gossip.gossip import Gossip
sawtooth-validator |   File "/usr/lib/python3/dist-packages/sawtooth_validator/gossip/gossip.py", line 729
sawtooth-validator |     << << << < HEAD
sawtooth-validator |      ^
sawtooth-validator | SyntaxError: invalid syntax
sawtooth-validator | bash: line 33: EOF: command not found
sawtooth-validator exited with code 127